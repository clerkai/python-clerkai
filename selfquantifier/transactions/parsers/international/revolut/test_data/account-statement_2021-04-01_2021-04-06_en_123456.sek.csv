Type,Product,Started Date,Completed Date,Description,Amount,Fee,Currency,State,Balance
CARD_PAYMENT,Current,2021-04-01 10:00:34,2021-04-02 12:11:40,Klarna *order Max Se S,-169.00,0.00,SEK,COMPLETED,549.18
EXCHANGE,Current,2021-04-03 09:50:29,2021-04-03 09:50:29,Exchanged to SEK,507.74,0.00,SEK,COMPLETED,1056.92
CARD_PAYMENT,Current,2021-04-03 09:51:21,2021-04-04 11:00:34,"St1 Taby, Bergt",-550.16,0.00,SEK,COMPLETED,506.76
EXCHANGE,Current,2021-04-06 08:05:38,2021-04-06 08:05:38,Exchanged to SEK,500.00,0.00,SEK,COMPLETED,1006.76
CARD_PAYMENT,Current,2021-04-06 08:06:29,2021-04-07 12:08:19,Easypark Ab,-35.40,0.00,SEK,COMPLETED,971.36
