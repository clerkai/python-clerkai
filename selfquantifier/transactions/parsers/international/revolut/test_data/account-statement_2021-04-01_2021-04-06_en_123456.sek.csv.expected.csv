Real Date,Bank Date,Payee,Bank Message,Amount,Currency,Balance,Foreign Currency,Foreign Currency Amount,Foreign Currency Rate,Original data,Raw Real Date,Raw Bank Date,Raw Payee,Raw Bank Message,Raw Amount,Raw Currency,Raw Balance,Raw Foreign Currency,Raw Foreign Currency Amount,Raw Foreign Currency Rate
2021-04-01 10:00:34,2021-04-02 12:11:40,,Klarna *order Max Se S,-169.00,SEK,549.18,,,,"{'Type': 'CARD_PAYMENT', 'Product': 'Current', 'Started Date': '2021-04-01 10:00:34', 'Completed Date': '2021-04-02 12:11:40', 'Description': 'Klarna *order Max Se S', 'Amount': -169.0, 'Fee': 0.0, 'Currency': 'SEK', 'State': 'COMPLETED', 'Balance': 549.18}",2021-04-01 10:00:34,2021-04-02 12:11:40,,Klarna *order Max Se S,-169.0,SEK,549.18,,,
2021-04-03 09:50:29,2021-04-03 09:50:29,,Exchanged to SEK,507.74,SEK,1056.92,,,,"{'Type': 'EXCHANGE', 'Product': 'Current', 'Started Date': '2021-04-03 09:50:29', 'Completed Date': '2021-04-03 09:50:29', 'Description': 'Exchanged to SEK', 'Amount': 507.74, 'Fee': 0.0, 'Currency': 'SEK', 'State': 'COMPLETED', 'Balance': 1056.92}",2021-04-03 09:50:29,2021-04-03 09:50:29,,Exchanged to SEK,507.74,SEK,1056.92,,,
2021-04-03 09:51:21,2021-04-04 11:00:34,,"St1 Taby, Bergt",-550.16,SEK,506.76,,,,"{'Type': 'CARD_PAYMENT', 'Product': 'Current', 'Started Date': '2021-04-03 09:51:21', 'Completed Date': '2021-04-04 11:00:34', 'Description': 'St1 Taby, Bergt', 'Amount': -550.16, 'Fee': 0.0, 'Currency': 'SEK', 'State': 'COMPLETED', 'Balance': 506.76}",2021-04-03 09:51:21,2021-04-04 11:00:34,,"St1 Taby, Bergt",-550.16,SEK,506.76,,,
2021-04-06 08:05:38,2021-04-06 08:05:38,,Exchanged to SEK,500.00,SEK,1006.76,,,,"{'Type': 'EXCHANGE', 'Product': 'Current', 'Started Date': '2021-04-06 08:05:38', 'Completed Date': '2021-04-06 08:05:38', 'Description': 'Exchanged to SEK', 'Amount': 500.0, 'Fee': 0.0, 'Currency': 'SEK', 'State': 'COMPLETED', 'Balance': 1006.76}",2021-04-06 08:05:38,2021-04-06 08:05:38,,Exchanged to SEK,500.0,SEK,1006.76,,,
2021-04-06 08:06:29,2021-04-07 12:08:19,,Easypark Ab,-35.40,SEK,971.36,,,,"{'Type': 'CARD_PAYMENT', 'Product': 'Current', 'Started Date': '2021-04-06 08:06:29', 'Completed Date': '2021-04-07 12:08:19', 'Description': 'Easypark Ab', 'Amount': -35.4, 'Fee': 0.0, 'Currency': 'SEK', 'State': 'COMPLETED', 'Balance': 971.36}",2021-04-06 08:06:29,2021-04-07 12:08:19,,Easypark Ab,-35.4,SEK,971.36,,,
